yarn.lock

```diff
@@ -4,44 +4,60 @@
 
 "@18f/identity-design-system@^7.1.0":
   version "7.1.0"
-  resolved "https://registry.yarnpkg.com/@18f/identity-design-system/-/identity-design-system-7.1.0.tgz#ddde006a4ee1371fed75dda6cd97ef0cc929331c"
+  resolved "https://registry.npmjs.org/@18f/identity-design-system/-/identity-design-system-7.1.0.tgz"
```

Comment:
It seems unexpected these are suddenly changing from `yarnpkg.com` to `npmjs.org`. Do you know why that is?

Reply:
I might suggest resetting `yarn.lock` to what's in `main`, rerunning `yarn install`, then a pass of `yarn-deduplicate`

```
git checkout main -- yarn.lock
yarn
yarn-deduplicate
```

When I do that, it restores `yarnpkg.com` URLs.

Reply:
Okay! That got messy. Thanks for the great review and catching this. When I changed `@babel/cli` from a dependencies to a devDependencies, I just added it to the list and ran `npm i`. I should be installing these package using yarn then via `yarn add @babel/cli --dev`

Reply:
> I might suggest resetting `yarn.lock` to what's in `main`, rerunning `yarn install`, then a pass of `yarn-deduplicate`
> 
> ```
> git checkout main -- yarn.lock
> yarn
> yarn-deduplicate
> ```
> 
> When I do that, it restores `yarnpkg.com` URLs.

I ran through these commands. The yarn lock file now uses yarnpkg.com and the file has a lot few changes.  Take a look- I will wait to merge until your approval. Thank you